sogs/model/user.py

```diff
@@ -230,6 +230,14 @@ def unban(self, *, unbanned_by: User):
         app.logger.debug(f"{unbanned_by} removed global ban on {self}")
         self.banned = False
 
+    def verify(self, *, message: bytes, sig: bytes):
+        """verify signature signed by this session id
+        return True if the signature is valid otherwise return False
+        """
+        return crypto.xed25519_verify(
+            sig, utils.decode_hex_or_b64(self.session_id[2:], size=32), message
```

Comment:
The `or_b64` here seems strange.  Could this just be `bytes.from_hex(self.session_id[2:])`?  That will throw a ValueError on non-hex; I assume xed25519_verify checks argument lengths and throws if not right.